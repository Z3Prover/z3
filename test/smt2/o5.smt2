; Copyright (c) 2015 Microsoft Corporation
(declare-const x Int)
(declare-const y Int)
(assert (<= x y))
(assert (<= y 5))
(maximize x)
(check-sat)
(get-objectives)