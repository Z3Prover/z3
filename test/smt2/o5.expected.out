sat
(objectives
 (x 5)
)
